src/Microsoft.Health.Fhir.Core/Features/Search/ContinuationToken.cs

```diff
@@ -0,0 +1,42 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Health.Fhir.Core.Features.Search
+{
+    /// <summary>
+    /// Represents the search continuation token.
+    /// </summary>
+    public class ContinuationToken
```

Comment:
This should instead be an array. Remember that while you are only adding support for sorting by one parameter, the spec allows sorting by any number. 
For that reason, I would make this an object array, the last entry of which would be the resource surrogate id.

Reply:
@johnstairs I converted the logic to handle an object array as you suggested, but kept the `ContinuationToken` class to handle all the little logic around getting and setting the values in the array.
I also added backward compatibility during parsing.